src/s2wasm.h

```diff
@@ -1342,6 +1346,40 @@ class S2WasmBuilder {
     }
   }
 
+  // Converts invoke wrapper names generated by LLVM backend to real invoke
```

Comment:
I see. Perhaps the name of the function can add `ForLLVM` or such? And clarify in the docs that this is LLVM specific.